package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "compile": "tsc index.ts test/spec.ts --module commonjs --noImplicitAny",
     "coverage": "cat ./coverage/lcov.info | coveralls",
-    "test": "istanbul cover _mocha",
+    "test": "istanbul cover node_modules/mocha/bin/_mocha",
```

Comment:
Sorry, I've just checked it and it doesn't work :(
It seems that the only option worked for me is `istanbul cover node_modules/mocha/bin/_mocha` as I initially proposed.